settings.gradle

```diff
@@ -64,6 +64,7 @@ buildCache {
 }
 
 include 'clients',
+    'clients:integration-tests',
```

Comment:
should we use `clients-integration-tests` to avoid potential conflict (https://github.com/gradle/gradle/issues/847) with `streams:integration-tests`?

Reply:
yes, good call 👍